Undo optimization attempts for the sqlite3 database (#2321)
pi-hole/FTL: src/CMakeLists.txt

# SQLITE_DEFAULT_FOREIGN_KEYS=1: This macro determines whether enforcement of foreign key constraints is enabled or disabled by default for new database connections.
# SQLITE_DQS=0: This setting disables the double-quoted string literal misfeature.
# SQLITE_ENABLE_DBPAGE_VTAB: Enables the SQLITE_DBPAGE virtual table. Warning: writing to the SQLITE_DBPAGE virtual table can very easily cause unrecoverably database corruption.

> # SQLITE_TEMP_STORE=0: This option sets the default value for the temp_store pragma to 0. This means that temporary tables and indices are always stored in memory. This is the default setting for SQLite.

Reviewer: ```suggestion
# SQLITE_TEMP_STORE=0: This option sets the default value for the temp_store pragma to 0. This means that temporary tables and indices are always stored on disk
```
